File: benchmark/corpus/04_off_by_one/d_empty_input.js
// Reads the last word of the input; there is no word, so position -1 is read.
function lastWord(words) {
  return words[words.length - 1]; // DANGER
}

const words = (await new Response(Deno.stdin.readable).text()).trim().split(" ").filter((p) => p.length > 0);
console.log("last word: " + lastWord(words));

File: benchmark/corpus/04_off_by_one/d_empty_input.py
# Reads the last word of the input; there is no word, so position -1 is read.
import sys


def last_word(words):
    return words[len(words) - 1]  # DANGER


words = [p for p in sys.stdin.readline().strip().split(" ") if p]
print("last word: " + last_word(words))
